src/mail/ngx_mail_auth_http_module.c

```diff
@@ -1228,6 +1228,8 @@ ngx_mail_auth_http_create_request(ngx_mail_session_t *s, ngx_pool_t *pool,
 #endif
 
     cscf = ngx_mail_get_module_srv_conf(s, ngx_mail_core_module);
+    ngx_uint_t client_port;
```

Comment:
Please declare variables at top of scope (watch variable ordering and alignment).